renderer/actions/types/editor-actions.ts

```diff
@@ -0,0 +1,50 @@
+import * as consts from '../../consts';
+
+export interface EditorActions{
+    editorUpdate: (newVal: string) => {
+        type: consts.EditorActionsTypes.EDITOR_UPDATE,
+        code: string
+    }
+
+    openFileSucceeded: (data: string, filepath: string) => {
+        type: consts.EditorActionsTypes.OPEN_FILE_SUCCEEDED,
+        code: string,
+        filepath: string,
+    }
+
+    saveFileSucceeded: (data: string, filepath: string) => {
+        type: consts.EditorActionsTypes.SAVE_FILE_SUCCEEDED,
+        code: string,
+        filepath: string
+    }
+
+    openFile: () => {
+        type: consts.EditorActionsTypes.OPEN_FILE
+    }
+
+    dragFile: (filepath: string) => {
+        type: consts.EditorActionsTypes.DRAG_FILE,
+        filepath: string
+    }
+
+    saveFile: (saveAs: boolean) => {
```

Comment:
In the original file, it gives a default parameter of False, but Typescript doesn't allow setting default parameter inside the interface. So I set the default parameter as False in the function at EditorActions.js instead.

Reply:
@HaileyJang Yes! The way you did it is correct, since we only declare type interfaces here, you can't declare default values in there, only the type which is `boolean` in this case.